docs/extend-graphql-schema.md

```diff
@@ -3,4 +3,568 @@ id: extend-graphql-schema
+
+const generalOpeningHours = [
+  "TUESDAY",
+  "WEDNESDAY",
+  "THURSDAY",
+  "FRIDAY",
+  "SATURDAY"
+].map(dayOfWeek => ({
+  day: dayOfWeek,
+  openingTimes: [{ from: "9:00", to: "12:30" }, { from: "14:00", to: "20:00" }]
+}));
+```
+
+Let’s analyze this code part by part to understand what is possible to achieve
+in a module.
+
+First of all, the exported resolver map contains a top-level query named
+`physicalStores`. This is not necessary (a module might as well extend existing
+types) and we recommend to try limiting as much as possible the number of
+top-level queries in your projects.
+
+```js
+export default {
+  Query: {
+    physicalStores: () => findAllStores()
+  }
+  // …
+};
+```
+
+The resolver function is pretty straightforward and needs no parameters: it will
+fetch all stores from a remote service (do not worry about the implementation
+yet) and return the data asynchronously using a Promise.
+
+The promise will resolve to a list of results matching the following format
+([view in your browser](https://nominatim.openstreetmap.org/?format=json&countrycodes=fr&addressdetails=1&q=foot%20locker&limit=10)):
+
+```json
+[
+  {
+    "place_id": "17302207",
+    "licence":
+      "Data © OpenStreetMap contributors, ODbL 1.0. https://osm.org/copyright",
+    "osm_type": "node",
+    "osm_id": "1528672001",
+    "boundingbox": ["48.858632", "48.858732", "2.347088", "2.347188"],
+    "lat": "48.858682",
+    "lon": "2.347138",
+    "display_name":
+      "Foot locker, Rue de Rivoli, Quartier des Halles, Quartier Les Halles, 1st Arrondissement, Paris, Ile-de-France, Metropolitan France, 75001, France",
+    "class": "shop",
+    "type": "shoes",
+    "importance": 0.20025,
+    "address": {
+      "shoes": "Foot locker",
+      "road": "Rue de Rivoli",
+      "neighbourhood": "Quartier des Halles",
+      "suburb": "Quartier Les Halles",
+      "city_district": "1st Arrondissement",
+      "city": "Paris",
+      "county": "Paris",
+      "state": "Ile-de-France",
+      "country": "France",
+      "postcode": "75001",
+      "country_code": "fr"
+    }
+  }
+  // …
+]
+```
+
+The schema defined earlier specified that `physicalStores` will return a list of
+`PhysicalStore` objects, which do not match the format returned by the remote
+service so far.
+
+```
+type Query {
+  "Our physical stores"
+  physicalStores: [PhysicalStore]
+}
+```
+
+In this implementation, we chose to use resolvers for converting data from the
+remote service into an object matching the `PhysicalStore` type definition. This
+time, by adding a resolver for each of its two fields:
+
+```js
+export default {
+  // …
+  PhysicalStore: {
+    coordinates: store => ({
+      latitude: store.lat,
+      longitude: store.lon
+    }),
+    openingHours: () => generalOpeningHours
+  }
+  // …
+};
+```
+
+The `coordinates` field is resolved by using the initial `store` information
+(first argument) and converting them into a structure matching our Schema. With
```

Comment:
What's the initial store information ? I'd mention that it's the one resulting from `findAllStores()`.